kolibri/plugins/management/assets/src/vue/user-page/user-edit-modal.vue

```diff
@@ -1,11 +1,11 @@
 <template>
 
   <div class="user-edit-modal">
-    <modal v-ref:modal btntext="Edit">
+    <modal v-on:open="clear()" v-ref:modal btntext="Edit">
 
       <h1 slot="header" class="header">Edit Account Info</h1>
 
-      <div @keyup.enter="editUser" v-if="!usr_delete && !pw_reset" slot="body">
+      <div @keyup.enter="editUser" v-if="!this.usr_delete && !this.pw_reset" slot="body">
 
         <div class="user-field">
           <label for="username">Full Name</label>:
```

Comment:
Label mixup here? `input id="name"` but `label for="username"`
@DXCanas @indirectlylit